src/components/organisms/Header/Header.tsx

```diff
@@ -32,17 +48,41 @@ export default function Header() {
     history.push('/newsfeed');
   };
   return (
-    <HeaderWrapper>
-      <Bubble onClick={goHome} style={{ margin: 30 }}>
-        <FontAwesomeIcon icon={faHome} size="5x" className="icon" />
-      </Bubble>
-      <HeaderText>Local Issues</HeaderText>
-      <Bubble style={{ margin: 30 }}>
-        <Avatar
-          src="http://stump.zackrose.net/images/avatar_sm.png"
-          size="3x"
+    <>
+      <HeaderWrapper>
+        <OnClickWrapper>
+          <FontAwesomeIcon
+            icon={faHome}
+            size="4x"
+            className="icon"
+            onClick={goHome}
+          />
+        </OnClickWrapper>
+        <HeaderText>Local Issues</HeaderText>
+        <OnClickWrapper>
+          <Avatar
+            src="http://stump.zackrose.net/images/avatar_sm.png"
+            size="3x"
+          />
+        </OnClickWrapper>
+      </HeaderWrapper>
+      <TabNavWrapper>
+        <TabNavigation
+          tabs={[
```

Comment:
You want to take this as input as opposed to hard coding it

Reply:
I am not quite sure what you mean for this one. Are you referring to the online styles?